tests/xtro-sharpie/MacCatalyst-CoreGraphics.todo

```diff
@@ -106,9 +111,9 @@
 !missing-pinvoke! CGDisplayModeGetTypeID is not bound
 !missing-pinvoke! CGDisplayModeGetWidth is not bound
 !missing-pinvoke! CGDisplayModeIsUsableForDesktopGUI is not bound
-!missing-pinvoke! CGDisplayModelNumber is not bound
 !missing-pinvoke! CGDisplayModeRelease is not bound
 !missing-pinvoke! CGDisplayModeRetain is not bound
+!missing-pinvoke! CGDisplayModelNumber is not bound
```

Comment:
not big deal - but it seems your sorting (Makefile) and the one done by C# differs wrt capital letters
and that might become noisy

Reply:
I'll see if the sort command can take the same sorting algo.